### PR TITLE
Updates Kinsta Deployment with SSR deployment

### PR DESCRIPTION
**Pull request by palmiak**
#### What kind of changes does this PR include?

- New or updated content

#### Description

- It adds a tutorial on how to deploy SSR Astro to Kinsta
- Also, it changes the structure of the Deploy to Kinsta page

### CODE REVIEW

**Review by ElianCodes — APPROVED** [review]
Looks good to me!

**Comment by ElianCodes** [comment]
Page of interest: https://deploy-preview-3030--astro-docs-2.netlify.app/en/guides/deploy/kinsta/

*just putting that up here in case anyone wants to read along*

**Review by ElianCodes — APPROVED** [review]
Looks very good! Approved by Talking and Doc'ing team!

Just a few comments

**Review by Jutanium — CHANGES_REQUESTED** [review]
Left some comments. Great work!

**Comment by ElianCodes** [comment]
Dan forgot to apply his changes; now that he did, when changes addressed, everything else looks fine!

File: src/content/docs/en/guides/deploy/kinsta.mdx [hunk 1]
```diff
@@ -29,12 +31,57 @@ Here are the necessary lines in your `package.json` file:
...
 }
 ```
 
+### SSR
+:::tip[Looking for an example?]
+Check out [the official Kinsta Application Hosting Starter project for Astro SSR](https://github.com/kinsta/hello-world-astro-ssr)!
+:::
+
+To host your project on **Kinsta Application Hosting**, you need to:
+- Include a `name` field in your `package.json`. (This can be anything, and will not affect your deployment.)
```
[the remark this answers is not in the document]
Comment (palmiak) [reply src/content/docs/en/guides/deploy/kinsta.mdx:46]: In theory, `name` is always required in the package.json file.package.json (https://docs.npmjs.com/creating-a-package-json-file#required-name-and-version-fields), also some buildpack environments (like the one that Kinsta uses) will throw an error when `name` isn`t there,

[the remark this answers is not in the document]
Comment (ElianCodes) [reply src/content/docs/en/guides/deploy/kinsta.mdx:46]: Little nitpick here, `name` is required, but only in the case you're actually wanting to publish the package on NPM

[the remark this answers is not in the document]
Comment (palmiak) [reply src/content/docs/en/guides/deploy/kinsta.mdx:46]: True, but an automated environment that is building from package.json file doesn't know that.

In general some buildpacks will throw an error that 'name' is missing. Kinsta requires it, I see that fly.io too (https://community.fly.io/t/fly-launch-couldnt-parse-package-json-error/6308) and probably Heroku too.

**Review by Jutanium — APPROVED** [review]

**Pull request merged**